src/platform/symbols.map

```diff
@@ -193,3 +193,12 @@ MIRPLATFORM_2.1 {
     mir::renderer::software::alloc_buffer_with_content*;
  };
 } MIRPLATFORM_2.0;
+
+MIR_PLATFORM_2.2 {
+ global:
+  extern "C++" {
+    mir::graphics::EGLExtensions::DebugKHR::DebugKHR*;
+    mir::graphics::EGLExtensions::DebugKHR::extension_or_null_object*;
+    mir::graphics::EGLExtensions::DebugKHR::maybe_debug_khr*;
+  };
+} MIRPLATFORM_2.1;
```

Comment:
That's weird (but not wrong). We've not released Mir 2.1.

In any case, I'm pretty sure we've broken mirplatform ABI by deleting stuff.